packages/patternfly-4/react-table/src/components/Table/utils/decorators/headerCol.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { IExtra, IFormatterValueType } from '../../Table';
+
+export const headerCol = (id: string = 'simple-node') => {
+  // tslint:disable-next-line:no-shadowed-variable
+  const headerCol = (value: IFormatterValueType, { rowIndex }:IExtra = {}) => {
+    const result = typeof value === 'object' ? value.title : value;
+    return {
+      component: 'th',
+      children: <div id={id + rowIndex}>{result}</div>
```

Comment:
We should probably avoid concatenating strings with `+` right? this could be:
```
<div id={`${id}${rowIndex}`}>
```
Not sure if it matters.

Reply:
updating!